.tekton/scanner-pull-request.yaml

```diff
@@ -0,0 +1,446 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/scanner?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && source_branch.contains("rhtap")
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: scanner
+    pipelines.appstudio.openshift.io/type: build
+  name: scanner-on-pull-request
+  namespace: rh-acs-tenant
+spec:
+  params:
```

Comment:
1. I once looked at `false` default value of `rebuild` param and concluded it's a premature optimization which prevents us from overwriting images with the same tags should we need to rebuild something.
Therefore, I suggest adding this to `params`

```
  - name: rebuild
    value: 'true'
```

2. Do you plan to enable gomod prefetching (`prefetch-input`) and `build-source-image` as part of this PR or subsequent PR(s)?

Reply:
1. d5c70398
2. May as well do it in this PR.  I'll keep this conversation open until it's successfully added.

Reply:
a3091e84fbc77a13e7db781